config/Map.yaml

```diff
@@ -119,3 +119,11 @@ mappings:
     newAPI: "apiVersion: rbac.authorization.k8s.io/v1\nkind: RoleBindingList"
     deprecatedInVersion: "v1.17"
     removedInVersion: "v1.22"
+  - deprecatedAPI: "apiVersion: admissionregistration.k8s.io/v1beta1\nkind: MutatingWebhookConfiguration"
+    newAPI: "apiVersion: admissionregistration.k8s.io/v1\nkind: MutatingWebhookConfiguration"
+    deprecatedInVersion: "v1.16"
+    removedInVersion: "v1.19"
+  - deprecatedAPI: "apiVersion: admissionregistration.k8s.io/v1beta1\nkind: ValidatingWebhookConfiguration"
+    newAPI: "apiVersion: admissionregistration.k8s.io/v1\nkind: ValidatingWebhookConfiguration"
+    deprecatedInVersion: "v1.16"
+    removedInVersion: "v1.19"
```

Comment:
As these APIs are not part of the [Helm install order list](https://github.com/helm/helm/blob/master/pkg/releaseutil/kind_sorter.go#L31-L66) then I am not sure they should be added by default to the mapping list. They can always be added on the fly.

Reply:
hi @hickeyma, how do you add them on the fly?

do we just need to update the `Map.yaml`?

Just wondering if you had any other recommendations.

Reply:
@Gearheads Sure, you can just add it to the file on the fly. Check out [API Mapping](https://github.com/hickeyma/helm-mapkubeapis#api-mapping) for more details.

Reply:
ok, understood.  thank you for the reply @hickeyma.